styles.js

```diff
@@ -1,6 +1,9 @@
 import { createGlobalStyle } from "styled-components";
 
 export default createGlobalStyle`
+@import url('https://fonts.googleapis.com/css2?family=Concert+One&display=swap');
```

Comment:
You should use ```next/font``` to implement Google Fonts in a save and compliant way. Check handout of session "React Styled Components 1":  https://github.com/neuefische/hh-web-24-4/blob/main/sessions/react-styled-components-1/react-styled-components-1.md

Reply:
Done, thank you :)